src/assets/_project/_blocks/layout/content/_content.scss

```diff
@@ -11,10 +11,16 @@
     color: $qg-dark-grey;
     font-weight: normal;
   }
+  blockquote{
```

Comment:
Please add this documentation for look and feel demo and to copy html structure.
https://qld-gov-au.github.io/web-template-release/styles.html

Reply:
Added